readme.md

```diff
@@ -8,46 +8,91 @@
 [![Backers][backers-badge]][collective]
 [![Chat][chat-badge]][chat]
 
-[**rehype**][rehype] plugin to wrap images in pictures.
+**[rehype][]** plugin to wrap images in pictures.
 
-## Install
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Install](#install)
+*   [Use](#use)
+*   [API](#api)
+    *   [`unified().use(rehypePicture[, options])`](#unifieduserehypepicture-options)
+*   [Types](#types)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## What is this?
+
+This package is a [unified][] ([rehype][]) plugin to change images (`<img>`)
+into pictures (`<picture>`).
+This lets you use a single image source in your content which is then
+automatically turned into a picture with several sources.
 
-This package is [ESM only](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c):
-Node 12+ is needed to use it and it must be `import`ed instead of `require`d.
+**unified** is a project that transforms content with abstract syntax trees
+(ASTs).
+**rehype** adds support for HTML to unified.
+**hast** is the HTML AST that rehype uses.
+This is a rehype plugin that changes images in the tree.
+
+## When should I use this?
+
+This plugin is useful when you have the same images in different file formats.
+For example, when you have a build step that turns `.jpg`s into `.webp`s, you
+can use this plugin to generate the markup for both.
```

Comment:
It is implied, but it may be worth explicitly stating.
This plugin expects images to already exist, it does not perform the transcoding itself.